### ISSUE TITLE
Example of 'remove' is wrong in README.md

### ISSUE THREAD

**Issue by jcampbell1710** [bozzltron/node-json-transform#44]
In README.md, you have this as an Advanced Example:
```var map = {
	list: 'items',
	item: {
		id: 'id',
...
		remove: ['unwanted']
	},
	defaults: {
		"missingData": true
	},
	operate: [
	....
};
```
The property 'remove' should be outside 'item', on same level as 'defaults'.

**Comment by bozzltron** [comment]
That is a correct.  Great catch.  Updating...

**Comment by bozzltron** [comment]
Fixed in 1.0.20
